Client/Frontend/Strings.swift

```diff
@@ -212,3 +212,21 @@ extension Strings {
     public static let switchToPBMKeyCodeTitle = NSLocalizedString("SwitchToPBMKeyCodeTitle", bundle: Bundle.shared, value: "Private Browsing Mode", comment: "Hardware shortcut switch to the private browsing tab or tab tray. Shown in the Discoverability overlay when the hardware Command Key is held down.")
     public static let switchToNonPBMKeyCodeTitle = NSLocalizedString("SwitchToNonPBMKeyCodeTitle", bundle: Bundle.shared, value: "Normal Browsing Mode", comment: "Hardware shortcut for non-private tab or tab. Shown in the Discoverability overlay when the hardware Command Key is held down.")
 }
+
+// Recent Searches
+extension Strings {
+    public static let recentSearchFavorites = NSLocalizedString("RecentSearchFavorites", bundle: Bundle.shared, value: "Favorites", comment: "Recent Search Favorites Section Title")
```

Comment:
Move everything to `BraveShared->BraveStrings`, and change bundle to `braveShared`